.github/workflows/pythonapp.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements/test.txt .
+        pip install  --no-cache-dir .[extra,tests,docs]
```

Comment:
no need for docs here. We will need it later I guess.

Reply:
yah i was kind of thinking ahead to when this is all automated, but no need for it now.